store/store.go

```diff
@@ -267,11 +269,13 @@ func authURL() string {
 	return "https://" + authLocation() + "/api/v2"
```

Comment:
You don't need /api/v2/, just https://login.ubuntu.com/ suffices if you're doing a GET. This is passed through to the app server anyway.

Reply:
I've left it at `/api/v2` as that's used elsewhere (changing it would require a minor refactor). Let me know if that's an issue.

Reply:
https://login.ubuntu.com/api/v2/ is a 404 :( But at least, the 404 *is* served by the app server so it does tell you whether the app server is responding; if it weren't, you'd get 50x errors. If that's enough for you then no problem from me.

Reply:
As this is a connectivity check and not a health check of the store (and we shouldn't be doing the latter), we only need to check that the host is reachable on port 443, not any particular endpoint, nor multiple checks to the same host (2 api.s.io calls).

Reply:
If you know a way of doing a single api.s.io call and also testing the cdn, speak up :-)

Reply:
I don't know, sorry but maybe @murraybd or @vorlonofportland does know.

Reply:
@mvo5 @pedronis no login.  The only snaps we're auto upgrading users to are free and publicly available and require no login, so the connectivity check should not require one either.